issue_11/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <div id="js-loading"></div><!-- /#js-loading -->
+    <ul id="js-ul"></ul><!-- /#js-ul -->
+
+    <script src="script.js"></script>
+</body>
```

Comment:
IMO
Sorry for the additional comment.🙇‍♀️
Declaring `<script>` with 'defer' in `<head> `allows the JS file to be loaded before HTML parsing is complete.
Try it if you like!
[<script> タグに async / defer を付けた場合のタイミング](https://qiita.com/phanect/items/82c85ea4b8f9c373d684)
```suggestion
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <script src="script.js" defer></script>
    <title>Document</title>
</head>
```

Reply:
First time I learned about it.:flushed:
I will use it in the future.
https://github.com/yoshikawa-akr/js-lessons/pull/11/commits/17c9baed15dbb14d406a39f6288f0d2b14efbc18